src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -20,7 +20,7 @@
 import seedu.address.model.book.Status;
 import seedu.address.model.book.exceptions.BookNotFoundException;
 import seedu.address.model.book.exceptions.DuplicateBookException;
-
+//@@author 592363789
```

Comment:
Add `//@@author` before [the `EditDescriptor` class](https://github.com/CS2103JAN2018-F14-B4/main/pull/111/commits/0dc97cb90b9abe89eb37b1c28b22b4b1d56189a6#diff-3462fc1118b9872387a01b41157b0402R136).